### PR TITLE
Data types

### PR DESCRIPTION
**Pull request by jonnathan-ls**
# [Data types](https://javascript.info/types)

## 📝 **Goal**

> Article update

## ☑ What was done?

- Pending text translation

### CODE REVIEW

**Comment by jonnathan-ls** [comment]
> Error: the article already translated in the Progress Issue #1 ⁉️
> https://github.com/javascript-tutorial/pt.javascript.info/pull/243#issuecomment-1157137217 [javascript-translate-bot](https://github.com/javascript-translate-bot)

There is still text that has not been translated.

**Comment by odsantos** [comment]
> > Error: the article already translated in the Progress Issue #1 interrobang
> > [#243 (comment)](https://github.com/javascript-tutorial/pt.javascript.info/pull/243#issuecomment-1157137217) [javascript-translate-bot](https://github.com/javascript-translate-bot)
> 
> There is still text that has not been translated.

Hi,

Will you translate the missing parts, including solution and task?
It would be helpful to review that as lesson, the whole translated article as well as the rest.

Trying to know your plan regarding to this lesson.

Thanks, Osvaldo.

**Review by renatocfrancisco — COMMENTED** [review]
In line 73, the english sentence still in the file.
Moreover, good job!

**Comment by jonnathan-ls** [comment]
Olá @renatocfrancisco,

Obrigado pela observação, não tinha percebido ❤️ 

Olá @odsantos, 

> Will you translate the missing parts, including solution and task?
> It would be helpful to review that as lesson, the whole translated article as well as the rest.

O PR contempla toda  tradução do artigo, com exceção da linha 73 como citado pelo Renato
Adicionei um novo commit com o ajuste.

**Comment by odsantos** [comment]
Hi @jonnathan-ls , sim. Logo que possivel. Obrigado.

**Comment by nazarepiedady** [comment]
@Peruibeloko, @jonnathan-ls, what is missing in this pull request?

**Comment by jonnathan-ls** [comment]
> ... what is missing in this pull request?

The review by two maintainers is missing, I will not contribute with the review, as the PR is precisely my own, there is no point in reviewing it myself

@nazarepiedady Could you add your review?

File: 1-js/02-first-steps/05-types/article.md
Comment (nazarepiedady) [inline 1-js/02-first-steps/05-types/article.md:182]: The meaning of `undefined` is **indefinido**, and when we talk about data types and values, it means **valor indefinido**.

To consider it as *valor não atribuído*, it should set as `null` that means absence of value. `undefined` means that the value and its type are not defined.

**Review by nazarepiedady — APPROVED** [review]
We could improve it more ahead, but for now, it is good enough to be approved.

**Comment by nazarepiedady** [comment]
@jonnathan-ls, now we need one more review. It should take some time because we are running under forces.

File: 1-js/02-first-steps/05-types/article.md
Comment (jonnathan-ls) [reply 1-js/02-first-steps/05-types/article.md:182]: It was maintained according to the original excerpt from the repository in English and when the initial translation occurred @nazarepiedady, see content in English https://github.com/javascript-tutorial/en.javascript.info/blob/master/1-js/02-first-steps/05-types/article.md#the-undefined-value

image

But I also consider the adjustment valid, being direct to say that it means `indefinido` makes more sense, although we would change the original content in English

Comment (nazarepiedady) [reply 1-js/02-first-steps/05-types/article.md:182]: Taking the things that way you are right, so let's keep the original approach of defining it.

**Comment by nazarepiedady** [comment]
@Peruibeloko, are you available to review this pull request?

File: 1-js/02-first-steps/05-types/article.md
Comment (Peruibeloko) [inline 1-js/02-first-steps/05-types/article.md:73]: This section is duplicated at line 75

File: 1-js/02-first-steps/05-types/article.md
Comment (Peruibeloko) [inline 1-js/02-first-steps/05-types/article.md:92]:
```
Você pode verificar a [tabela de compatibilidade do *MDN* sobre BigInt]
```

ou 

```
Você pode verificar a [tabela de compatibilidade do BigInt na *MDN*]
```

**Review by Peruibeloko — CHANGES_REQUESTED** [review]
Duas mudanças rápidas e está aprovado

**Comment by nazarepiedady** [comment]
@jonnathan-ls, could you advance with this?

File: 1-js/02-first-steps/05-types/article.md
Comment (jonnathan-ls) [reply 1-js/02-first-steps/05-types/article.md]: Removed

File: 1-js/02-first-steps/05-types/article.md
Comment (jonnathan-ls) [reply 1-js/02-first-steps/05-types/article.md]: Done

**Comment by jonnathan-ls** [comment]
> ... could you consider the reviews did?

/done

I applied the requests noted by @Peruibeloko, see commit https://github.com/javascript-tutorial/pt.javascript.info/pull/243/commits/4a7029ed5e88cb33e49b5c952a71c5f0987f6a8a. And I also added pending content that I noticed when accessing the article from the official English repository [bigint-type session](https://github.com/javascript-tutorial/en.javascript.info/blob/master/1-js/02-first-steps/05-types/article.md#bigint-bigint-type), see commit https://github.com/javascript-tutorial/pt.javascript.info/pull/243/commits/7cafcbb3498aa3c5eb433a6104f8d1cd45b84ec2

**Review by nazarepiedady — APPROVED** [review]
It seems good enough to be approved.

**Review by Peruibeloko — APPROVED** [review]
LGTM

**Pull request merged**
